frontend/micro-ui/web/micro-ui-internals/packages/modules/hrms/src/components/SearchUserForm.js

```
@@ -351,31 +411,49 @@ const SearchUserForm = ({ uniqueTenants, setUniqueTenants, roles, setUniqueRoles
     setUniqueTenants(() => listOfUniqueTenants);
     setUniqueRoles(() => data?.roles?.filter((row) => row.code)?.map((role) => role.code));
   };
+  const [divisionTree, setDivisionTree] = useState(null);
+
+  useEffect(() => {
+    if (userData) {
+      const zoneC = userData[0].zoneCode;
+      const circleC = userData[0].circleCode;
+      const divisionC = userData[0].divisionCode;
+
+      if (tree && tree[zoneC] && tree[zoneC][circleC]) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)